implementations/rust/ockam/ockam_node/src/node.rs

```diff
@@ -60,7 +59,7 @@ impl NodeBuilder {
                 vec![],
             ),
             None,
-            Arc::new(Mutex::new(HashMap::new())),
+            Arc::new(Mutex::new(Default::default())),
```

Comment:
Have you tried `Default::default()` instead of this whole line? It compiler able to figure it out?

Reply:
No, that didn't work unfortunately.

Reply:
It works with `RwLock`.

Reply:
Works with `Mutex` as well, just in case